pkg/apis/v1alpha1/serviceimport.go

```diff
@@ -23,6 +23,8 @@ import (
 
 // +genclient
 // +kubebuilder:object:root=true
+// +kubebuilder:storageversion
```

Comment:
Shouldn't the new _v1alpha2_ be the stored version?

Reply:
The problem is that the CoreDNS plugin or anything watching mcs api ressources would need to support v1alpha2 explicitly or that we have the webhooks already deployed/figured out. So my plan would be to do this after CoreDNS supports v1alpha2, see this https://github.com/kubernetes-sigs/mcs-api/pull/85#issuecomment-2514593992